resources/views/model.blade.php

```diff
@@ -40,11 +44,10 @@ class {{ $modelBaseName }} extends Model
 @endif
 @endforeach
 @endif
-
     @if (!is_null($tableName))protected $table = '{{ $tableName }}';
-    @endif
 
-    @if ($fillable)protected $fillable = [
+    @endif
+@if ($fillable)protected $fillable = [
     @foreach($fillable as $f)
     "{{ $f }}",
```

Comment:
It is possible to change double quotes to single quotes, my editor hates me for that.